model/fv_regional_bc.F90

```diff
@@ -3975,12 +4214,12 @@ subroutine remap_dwinds_regional_bc(Atm                              &
 
      do k=1,km+1
         do i=is_v,ie_v
-           pe0(i,k) = ak0(k) + bk0(k)*0.5*(psc(i-1,j)+psc(i,j))
+           pe0(i,k) = ak0(k) + bk0(k)*psc(i,j)
```

Comment:
I have the same question as @kaiyuan-cheng - why are you removing the average ,which provides the pressure at the cell edge values where the winds are defined, in favor of the cell mean value defined at the cell center?

Reply:
The i-1 value is outside the boundary in some places.